test/HashBase.test.ts

```diff
@@ -9,23 +9,34 @@ import { BaseType, HashBaseConfig } from '../src/types'
 
 interface TestData {
   hash: string
-  input: string
-  output: string
+  name: string
+}
+interface TestDataFixture {
+  [partitionName: string]: TestData[]
 }
 
 describe('HashBase baselet', function () {
   const memlet = makeMemlet(makeMemoryDisklet())
   let hashbaseDb: HashBase<TestData>
```

Comment:
This is out of scope for your changes, but each test suite is supposed to be standalone. Mutating a single `hashbaseDb` from test-to-test means a failure earlier, like in the "insert" test, will also cause fake failures in the "query" test, because the state is wrong. Each test should create & initialize its own unique HashBase instance.

Reply:
Yeah, this is inherited. I can create a task for this, but which project should such a task go (backlog?)